### ISSUE TITLE
Rename Time Distributed dense/softmax

### ISSUE THREAD

**Issue by patyork** [fchollet/keras#46]
I'm thinking that "Temporal" is a better prefix that "Time Distributed". It fits most papers better, and means essentially the same thing.

**Comment by patyork** [comment]
Any opinion on this?

**Comment by fchollet** [comment]
I personally have no opinion. Maybe "time distributed" is less ambiguous, as "distributed" implied that you are doing the same thing over different timesteps, rather than doing a temporal operation (like a temporal convolution, for instance). 
But maybe that's also clear with "temporal". 

It would be interesting to gather a few opinions from users, about which name sounds more natural and explicit.
